apps/react-workshop/src/NotificationMarker.stories.tsx

```diff
@@ -59,58 +58,54 @@ export const WithButtons = () => {
 export const Pulsing = () => {
   return (
     <>
-      <IconButton styleType='borderless'>
+      <IconButton label='Notifications' styleType='borderless'>
         <NotificationMarker status='negative' pulsing>
           <SvgNotification />
         </NotificationMarker>
       </IconButton>
-
-      <NotificationMarker status='negative' pulsing>
-        <Text style={{ fontWeight: '700' }}>URGENT</Text>
-      </NotificationMarker>
     </>
   );
 };
 
 export const Status = () => {
   return (
     <>
-      <NotificationMarker status='primary'>Primary</NotificationMarker>
-      <IconButton styleType='borderless'>
+      <Text>Primary</Text>
```

Comment:
If you're changing the stories, the images will need to be updated.

Alternatively, revert the stories back. The stories are for internal testing and do not need to always match the documentation.

Reply:
> If you're changing the stories, the images will need to be updated.

Yes, I was in the process of running the tests and approving them. Done (https://github.com/iTwin/iTwinUI/pull/1999/commits/0c101d4587b3af6a021be2e464f1de3e0e64c885).

> Alternatively, revert the stories back. The stories are for internal testing and do not need to always match the documentation.

Some of the changes are necessary such as adding the `label` to the `IconButton`s. So, I think I'll keep the other optional changes too, since I've already made those changes.